### ISSUE TITLE
Creating a data set of 3d landbarks

### ISSUE THREAD

**Issue by bchapuis** [wipfli/3d-landmarks#2]
Have you considered using openstreetmap and wikidata to create a dataset of 3d landmarks? This is something we could discuss next week at the OGC code sprint if you are interested.

![Screenshot 2023-04-22 at 14 38 17](https://user-images.githubusercontent.com/238747/233785248-ff313162-cf0b-4b7f-8f88-22c93d6cdfec.png)

![Screenshot 2023-04-22 at 14 38 22](https://user-images.githubusercontent.com/238747/233785256-49285bd8-3ee2-4e45-9033-9e6acc62a890.png)

**Comment by wipfli** [comment]
Yeah let's discuss this next week. More and more 3d data will become available I think with time.
